src/main/java/no/nav/foreldrepenger/regler/uttak/fastsetteperiode/FastsettePeriodeRegel.java

```diff
@@ -205,7 +206,14 @@ private FastsettePeriodeUtfall ikkeOppfyltUT1166() {
                 "Er hele perioden innenfor maksimalgrense for foreldrepenger?")
                 .hvis(new SjekkOmPeriodenErEtterMaksgrenseForUttak(konfigurasjon),
                         IkkeOppfylt.opprett("UT1085", IkkeOppfyltÅrsak.UTTAK_ETTER_3_ÅRSGRENSE, false, false))
-                .ellers(sjekkOmUttaksperiodenEtterSøkersDødsdato());
+                .ellers(sjekkOmPeriodeEtterNyStønadsperiode());
+    }
+
+    private Specification<FastsettePeriodeGrunnlag> sjekkOmPeriodeEtterNyStønadsperiode() {
+        return rs.hvisRegel(SjekkOmPeriodenErEtterNyStønadsperiode.ID, "Er uttaksperioden etter start av ny stønadsperiode?")
```

Comment:
Har begynt å legge inn beskrivelser inn i klassen SjekkOmPeriodenErEtterNyStønadsperiode.BESKRIVELSE,
så slipper vi duplisering nårvi gjenbruker sjekken andre steder.

Reply:
Done. Men hele id-politikken er rar på disse anonyme hvis-reglene. 
Legg merke til @RuleDocumentation(SjekkOmPeriodenErEtterNyStønadsperiode.ID) 

For SVP lagde jeg en hvis.hvis.hvis.ellers kjede og kalte kjeden for "Fastsett uttak" ...